patches/tModLoader/Terraria/Item_tML.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Terraria.ModLoader.IO;
+
+namespace Terraria
+{
+	public partial class Item : TagSerializable
```

Comment:
To be honest I don't like the "_tML" suffix either, but I couldn't come up with anything better. Could possibly split it out into another folder and name it just "ClassName.cs", but I'm not sure whether I like the idea of partial classes being completely separated in different folders.